.github/workflows/release.yml

```diff
@@ -107,16 +110,16 @@ jobs:
         with:
           # match pr comment like with changesets-snapshot-action from the guild's shared-config
           GITHUB_TOKEN: ${{ secrets.BOT_GITHUB_TOKEN }}
-          header: snapshot-release-docker-image
+          header: snapshot-release-${{ matrix.runtime == 'Bun' && 'bun-' || '' }}docker-image
           message: |
-            ### 🚀 Snapshot Release (Docker Image)
+            ### 🚀 Snapshot Release (${{matrix.runtime}} Docker Image)
 
             The latest changes of this PR are available as image on GitHub Container Registry (based on the declared `changesets`):
             ```
-            ghcr.io/graphql-hive/gateway:${{ fromJSON(steps.ver-gateway.outputs.result).version }}
+            ghcr.io/graphql-hive/gateway:${{ fromJSON(steps.ver-gateway.outputs.result).version }}${{ matrix.runtime == 'Bun' && '.bun' || '' }}
```

Comment:
Convention is to use a dash for different runtimes/OS. See [node dockerimages](https://hub.docker.com/_/node) (node-alpine, node-bookworm, etc.)

The reason dash is preferred is to avoid confusion with versions. For example, `gateway:1.2.bun` makes "bun" a patch version.

Reply:
Done!